### DISCUSSION TITLE
拖拽的节点粘着鼠标

### DISCUSSION THREAD

**Discussion by lq9958** [didi/LogicFlow#1832] [Q&A]
# package info    

> @logicflow/core-2.0.3
> vue-3.4.31

需要点击一下才能创建节点，并且节点的生命周期执行了两次

https://github.com/user-attachments/assets/36e8d342-f285-4e7c-99b8-eb71d9f83c69

**Comment by yangzilucky** [comment]
拖拽的元素加上 @mousedown.prevent="$_dragNode(item)"试一下，prevent是重点
